src/Runner.Worker/action_yaml.json

```diff
@@ -120,6 +137,21 @@
             ],
             "string": {}
         },
+        "composite-output-value": {
+            "context": [
```

Comment:
we might want to double-check what is allowed.
Ex: as a composite action author, how do i use `needs`, `secrets` and `steps`?

Reply:
also, remove `secrets`, `needs`, all the condition func from `composite-steps`

Reply:
tl;dr clean up secrets + needs from composite-steps and compomsite-output-value.